.eslintrc

```diff
@@ -40,6 +41,12 @@
     "import/no-extraneous-dependencies": 0,
     "prettier/prettier": ["error", {
       "singleQuote": true
+    }],
+    "graphql/template-strings": ['error', {
+      // Import default settings for your GraphQL client. Supported values:
+      // 'apollo', 'relay', 'lokka', 'literal'
+      env: 'literal'
```

Comment:
https://github.com/apollographql/eslint-plugin-graphql#graphql-literal-files

```
This plugin also lints GraphQL literal files ending on .gql or .graphql. In order to do so set env to 'literal' in your .eslintrc.js and tell eslint to check these files as well.
```

at first I thought `apollo` too, but with it eslint doesnt check .gql files